src/pe/import.rs

```diff
@@ -344,34 +345,54 @@ impl<'a> ImportData<'a> {
             "import_directory_table_rva {:#x}",
             import_directory_table_rva
         );
-        let offset =
-            &mut utils::find_offset(import_directory_table_rva, sections, file_alignment, opts)
-                .ok_or_else(|| {
-                    error::Error::Malformed(format!(
+
+        let offset_opt = utils::find_offset(import_directory_table_rva, sections, file_alignment, opts);
+
+        if offset_opt.is_none() {
+            println!(
```

Comment:
no printlns; must log; i think you should check the parse options here as well, as seems like this is pretty malformed if we fail here?